README.md

```diff
@@ -52,7 +52,7 @@ performance, small size and all the features listed below.
   * A rich set of hooks available, both per vnode and globally for modules,
     to hook into any part of the diff and patch process.
   * Splendid performance. Snabbdom is among the fastest virtual DOM libraries
-    in the [Virtual DOM Benchmark](http://vdom-benchmark.github.io/vdom-benchmark/).
+    in the Virtual DOM Benchmark.
```

Comment:
```
* Splendid performance. Snabbdom is among the fastest virtual DOM libraries.
```

I feel the link should be removed completely.

Reply:
@ mightyiam fell so, removed.